business_register/converter/declaration.py

```diff
@@ -79,12 +79,22 @@ def __init__(self):
             'Юридична особа, зареєстрована в Україні': BaseRight.UKRAINE_LEGAL_ENTITY,
             'Юридична особа, зареєстрована за кордоном': BaseRight.FOREIGN_LEGAL_ENTITY,
         }
-        # TODO: delete ENIGMA, keys
-        self.ENIGMA = {'1', 'j'}
         self.DECLARANT = '1'
         self.OTHER_PERSON = 'j'
         self.UKRAINE = Country.objects.get(name='ukraine')
-        self.keys = set()
+        self.UNIDENTIFIED_PEP, _ = Pep.include_deleted_objects.get_or_create(
+            first_name='',
+            middle_name='',
+            last_name='UNIDENTIFIED_PEP',
+            fullname='UNIDENTIFIED_PEP',
+            is_dead=True,
+            deleted_at='2021-01-01 10:05:26.268544'
+        )
```

Comment:
`'UNIDENTIFIED_PEP'` - should be constant
```suggestion
        self.UNIDENTIFIED_PEP = Pep.get_unidentified_pep()
```

Reply:
done